docs/javascript/programming-basics/conditionals.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 6
+---
+
+# Condicionais
+
+## O que são condicionais?
+
+Condicionais são estruturas de controle que permitem que o programa execute um bloco de código somente se uma determinada condição for verdadeira.
+
+## Como usar condicionais?
+
+### Quais valores um condicional pode receber?
+
+Qualquer valor [booleano ou expressão que retorne um valor booleano](pathname:///docs/javascript/programming-basics/data-types) pode ser usado em uma estrutura condicional.
+
+```js title="if.js"
+if (true) {
+  console.log('if(true) foi executado');
+}
+if (false) {
+  console.log('if(false) foi executado');
+}
+
+let a = true;
+if (a) {
+  console.log('if(a) foi executado');
+}
+
+let x = 5;
+let y = 10;
+let compare = x > y;
+if (a || compare) {
+  console.log('if(a || compare) foi executado');
+}
+if (a && compare) {
+  console.log('if(a && compare) foi executado');
+}
+```
+
+```bash title="if.js output"
+if(true) foi executado
+if(a) foi executado
+if(a || compare) foi executado
+```
+
+> Qualquer valor que não seja `0`, `null`, `undefined`, `false` ou `NaN` é considerado `true` e qualquer valor que seja `0`, `null`, `undefined`, `false` ou `NaN` é considerado `false`.
+
+## Múltiplas condições de uma mesma variável
```

Comment:
```suggestion
## Múltiplas condições
```

In that section, I think we should go a little deeper into the "if" and "else" terms' definitions, translating them, also we should talk about why using `switch/case`. 
- What are the benefits? 
- Why it can be faster?